metadata-ingestion/src/datahub/sql_parsing/sql_parsing_aggregator.py

```diff
@@ -1383,7 +1390,16 @@ def _gen_lineage_for_downstream(
                             if self.can_generate_query(query_id)
                             else None
                         ),
-                        confidenceScore=queries_map[query_id].confidence_score,
+                        confidenceScore=query.confidence_score,
+                        transformOperation=(
```

Comment:
Is there a reason we format as a string and not convert it into a JSON object to make it parsable by machine?

Reply:
I believe this is shown in the UI as-is